version.gradle

```diff
@@ -29,7 +29,7 @@
  * already in the root directory.
  */
 
-final def SPINE_VERSION = '1.0.1'
+final def SPINE_VERSION = '1.0.2'
```

Comment:
Currently, we don't have `time`, `core`, `client`, and `server` in this version. The build will not pass without those dependencies. 
Let's discuss in person.